src/json-patch-duplex.ts

```diff
@@ -524,6 +525,65 @@ module jsonpatch {
     return true;
   }
 
+  
+  /**
+     * Apply a single json-patch on an object tree
+     * Returns the result object.
+     */
+  export function applyPatch<T>(tree: T, patch: Patch<any>): T;
+  export function applyPatch<T>(tree: T, patch: Patch<any>, validate: boolean, touchOriginalTree: boolean): T;
+  export function applyPatch<T>(tree: T, patch: Patch<any>, validate = false, touchOriginalTree = true): T {
```

Comment:
No, because that is just the implementation. This signature is overloaded, so the two signatures above the implementation that just end with a semicolon define the signature. Overloading is needed here because without the first signature the function would not work as reducer.